mitodl/mitx-theme
# Remove lms-main-v2.scss due to compilation error

#### What are the relevant tickets?
fixes https://github.com/mitodl/mitx-theme/issues/120

#### What's this PR do?
It removes `lms-main-v2.scss` as it's not being used in the theme and also marked as deprecated in edx-platform.

#### How should this be manually tested?
Anyone can simply try to configure/install this theme to see if it's working as expected or not. Previously it was giving compile-time errors.

#### Where should the reviewer start?
The reviewer may read https://github.com/edx/edx-platform/blob/master/docs/guides/frontend/styling.rst
